EventFilter/HGCalRawToDigi/src/HGCalUnpacker.cc

```diff
@@ -152,12 +152,21 @@ uint8_t HGCalUnpacker::parseFEDData(unsigned fedId,
             << "), got 0x" << econd_headers[0] << ".";
         return UNPACKER_STAT::WrongECONDHeader;
       }
+      // Compute ECON-D trailer CRC
+      bool crcvalid(true);
+      const auto econd_payload_length = ((econd_headers[0] >> ECOND_FRAME::PAYLOAD_POS) & ECOND_FRAME::PAYLOAD_MASK);
+      crcvalid = hgcal::econdCRCAnalysis(ptr, 0, econd_payload_length);
```

Comment:
`bool crcvalid = ...` here would avoid the need for the previous definition two lines above.

Reply:
`econd_payload_length` is used in more lines below, so I kept the definition, but removed L156.